smithy-aws-protocol-tests/model/awsJson1_1/services/machinelearning.smithy

```diff
@@ -0,0 +1,181 @@
+$version: "1.0"
+
+namespace com.amazonaws.machinelearning
+
+use aws.api#service
+use aws.auth#sigv4
+use aws.protocols#awsJson1_1
+use smithy.test#httpRequestTests
+
+@service(
+    sdkId: "Machine Learning",
+    arnNamespace: "machinelearning",
+    cloudFormationName: "MachineLearning",
+    cloudTrailEventSource: "machinelearning.amazonaws.com",
+    endpointPrefix: "machinelearning",
+)
+@sigv4(
+    name: "machinelearning",
+)
+@awsJson1_1
+@title("Amazon Machine Learning")
+@xmlNamespace(
+    uri: "http://machinelearning.amazonaws.com/doc/2014-12-12/",
+)
+service AmazonML_20141212 {
+    version: "2014-12-12",
+    operations: [
+        Predict,
+    ],
+}
+
+@httpRequestTests([
+    {
+        id: "MachinelearningPredictEndpoint",
+        documentation: """
+            MachineLearning's api makes use of generated endpoints that the
+            customer is then expected to use for the Predict operation. Having
+            to alter the endpoint for a specific operation would be cumbersome,
+            so an AWS client should be able to do it for them.""",
+        protocol: awsJson1_1,
+        method: "POST",
+        uri: "/",
+        resolvedEndpoint: "custom.example.com",
```

Comment:
It looks like this got moved to `resolvedHost` for the naming.

Should this take a base `host` to indicate that it's being fully overridden?

Reply:
hmm yeah why is this passing